apps/files/src/mixins.js

```diff
@@ -1,6 +1,5 @@
 import filesize from 'filesize'
 import moment from 'moment'
-import FileUpload from './FileUpload.js'
```

Comment:
can we delete FileUpload.js?

Reply:
yes, when I'm done. See TODO list in top post